frontend/src/pages/destructionlist/DestructionListDetail.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  AttributeTable,
+  Body,
+  CardBaseTemplate,
+  Column,
+  Grid,
+  H1,
+  ObjectData,
+  P,
+} from "@maykin-ui/admin-ui";
+import { ActionFunctionArgs } from "@remix-run/router/utils";
+import React from "react";
+import { redirect, useLoaderData } from "react-router-dom";
+
+import { loginRequired } from "../../lib/api/loginRequired";
+import { request } from "../../lib/api/request";
+import { User } from "../../lib/api/reviewers";
+import { AssigneesEditable } from "./Assignees";
+import { getZakenData } from "./DestructionListCreate";
+import "./DestructionListDetail.css";
+import { DestructionListItems } from "./DestructionListItems";
+import {
+  DestructionListData,
+  DestructionListDetailContext,
+  DestructionListItemUpdate,
+  DestructionListUpdateData,
+} from "./types";
+
+// Todo: should this come from the backend?
+const STATUS_MAPPING: { [key: string]: string } = {
+  in_progress: "In progress",
+  processing: "Processing",
+  completed: "Completed",
+};
+
+function formatUser(user: User) {
```

Comment:
The landing page doesn't do this either right now, but I think we should think about combining some of the things here like the `STATUS_MAPPING` and the `formatUser()` since the landing page also uses them (and possibly others later on). For example a function `formatUser()` that takes in some options to customize the output would be nice, and a central place to get the status mappings from. 
I think we can ignore it for now though

Reply:
I moved the status mappings to a constants.ts files and the format user function to a utils file. Then it can be more easily reused by the landing page. Then we can expand the options once we have a better idea of what we need!